code/modules/food_and_drinks/food/foods/candy.dm

```diff
@@ -644,3 +644,12 @@
 	icon_state = "toolerone"
 	filling_color = "#7D5F46"
 	goal_difficulty = FOOD_GOAL_NORMAL
+
+/obj/item/food/candy/chocolate_bar
+	name = "Delux Chocolate-bar"
+	desc = "Crafted from premium cacao and infused with quantum nutrients, this luxury chocolate bar is wrapped in nanofoil for ultimate freshness. Experience the essence of culinary supremacy with every bite."
+	icon_state = "choco"
+	filling_color = "#7D5F46"
+	list_reagents = list("chocolate" = 5, "sugar" = 10)
+	tastes = list("chocolate" = 3)
```

Comment:
```suggestion
	tastes = list("chocolate" = 1)
```
Yeah, no reason for it not to be one when the list only has one taste.

Reply:
All done !